scene/resources/2d/convex_polygon_shape_2d.cpp

```diff
@@ -93,6 +93,10 @@ Vector<Vector2> ConvexPolygonShape2D::get_points() const {
 	return points;
 }
 
+bool ConvexPolygonShape2D::contains_point(const Vector2 &p_point) const {
+	return false; //Requires rewriting base class to include normals of points
```

Comment:
```suggestion
	return false; // Requires rewriting base class to include normals of points.
```
Is this a TODO comment, or just something unfeasible?

Reply:
It could be a TODO comment for the future. It's not entirely infeasible, but it was outside the scope of what I was doing.